application/src/test/resources/standalone/config/build-config.json

```diff
@@ -85,7 +85,8 @@
     },
     "BIKE": {
       "maxTransferDuration": "30m",
-      "carsAllowedStopMaxTransferDuration": "3h"
+      "carsAllowedStopMaxTransferDuration": "3h",
+      "bikesAllowedStopMaxTransferDuration": "1h"
```

Comment:
Can you add `"disableDefaultTransfers": true` here because without it using `bikesAllowedStopMaxTransferDuration` does not make sense. Just in case someone uses this as an example.

Also can you add this to the `transferRequests`, so the example makes more sense:
```
{ "modes": "BICYCLE" },
{ "modes": "CAR" }
```